<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 250, 254);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="buttons_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>106</width>
          <height>700</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>106</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
	background-color: rgb(31, 149, 239);
}
QPushButton{
	color:white;
	text-align:center;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="CNN_EEG_Button">
             <property name="text">
              <string>CNN for EEG</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="CNN_MRI_Button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>CNN for MRI</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="GAN_MRI_Button">
             <property name="text">
              <string>GAN for MRI</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>382</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="dbButton_2">
           <property name="text">
            <string>Database Admin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="switchSceneBtn">
           <property name="text">
            <string>USER View</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exitButton">
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QWidget" name="main_widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>720</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QPushButton" name="folder_explore_3">
                 <property name="text">
                  <string>Choose data folder</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="info_dump_2">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QWidget" name="widget_3" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <item>
                   <spacer name="verticalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_13">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_4">
                      <item>
                       <widget class="QLabel" name="epochs_3">
                        <property name="minimumSize">
                         <size>
                          <width>120</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Epochs</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QSpinBox" name="textEdit_epochs_2">
                        <property name="minimum">
                         <number>1</number>
                        </property>
                        <property name="maximum">
                         <number>9999999</number>
                        </property>
                        <property name="value">
                         <number>1</number>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_3">
                      <item>
                       <widget class="QLabel" name="batch_size">
                        <property name="minimumSize">
                         <size>
                          <width>120</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Batch size</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QSpinBox" name="textEdit_batch_size_2">
                        <property name="minimum">
                         <number>1</number>
                        </property>
                        <property name="maximum">
                         <number>10000</number>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <item>
                       <widget class="QLabel" name="learning_rate">
                        <property name="minimumSize">
                         <size>
                          <width>120</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Learning rate</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QDoubleSpinBox" name="textEdit_learning_rate_2">
                        <property name="decimals">
                         <number>4</number>
                        </property>
                        <property name="minimum">
                         <double>0.000100000000000</double>
                        </property>
                        <property name="maximum">
                         <double>1.000000000000000</double>
                        </property>
                        <property name="singleStep">
                         <double>0.000100000000000</double>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <item>
                       <widget class="QLabel" name="Test_size">
                        <property name="minimumSize">
                         <size>
                          <width>120</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Test size</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QDoubleSpinBox" name="textEdit_test_size">
                        <property name="decimals">
                         <number>1</number>
                        </property>
                        <property name="minimum">
                         <double>0.100000000000000</double>
                        </property>
                        <property name="maximum">
                         <double>0.900000000000000</double>
                        </property>
                        <property name="singleStep">
                         <double>0.100000000000000</double>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <widget class="QLabel" name="desc_label_2">
                   <property name="text">
                    <string>Description</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTextEdit" name="model_description_2"/>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_12">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_15">
                     <item>
                      <widget class="QPushButton" name="startButton_2">
                       <property name="text">
                        <string>Start</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="status_label_2">
                       <property name="text">
                        <string>STATUS:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="stopButton_2">
                       <property name="text">
                        <string>Stop</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QLabel" name="more_info_dump_2">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_16">
                     <item>
                      <widget class="QPushButton" name="save_db_2">
                       <property name="text">
                        <string>Save model to db</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="db_status_2">
                       <property name="text">
                        <string>STATUS:</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="progressBar_2">
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="window_name_2">
             <property name="text">
              <string>CNN for MRI</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>640</width>
            <height>640</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget{
	background-color: rgb(31, 149, 239);
}</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="plotLabel_CNN_2">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>icons/icon_loop.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
